plugin/storage/es/requirements.txt

```diff
@@ -0,0 +1,56 @@
+#
```

Comment:
Where is this file used in the build?

Reply:
Right, sorry, this should have been part of the Dockefiles. PR updated.